.env.defaults

```diff
@@ -1,4 +1,4 @@
 EPOCH_DURATION=1440
```

Comment:
@rokso Does the epoch duration need to be changed along with the amount?

Reply:
We can keep it as is. Epoch is being used for sampling data. Actual reward duration comes from start and end blocks